nilearn/datasets/atlas.py

```diff
@@ -811,8 +805,9 @@ def fetch_coords_power_2011(legacy_format=True):
 
 
 @fill_doc
-def fetch_atlas_smith_2009(data_dir=None, mirror='origin', url=None,
-                           resume=True, verbose=1):
+def fetch_atlas_smith_2009(data_dir=None, url=None, resume=True,
```

Comment:
```suggestion
def fetch_atlas_smith_2009(data_dir=None, mirror='origin', url=None, resume=True,
```

Reply:
Hello @bthirion, as all the other methods have the same starting signature (data_dir/url/ resume/verbose), wouldn't be better to leave the mirror parameter to the end of the signature?

Reply:
Indeed, you're right